src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const value = 40;
+  const minimumDiscount = 20;
+  const maximumDiscount = 50;
+  const basePrice = days * value;
+
+  if (days <= 2) {
+    return basePrice;
+  } else if (days >= 3 && days <= 6) {
+    return basePrice - minimumDiscount;
```

Comment:
The condition 'days >= 3 && days <= 6' can be simplified. Since the previous return statement will exit the function if the days are less than 3, you only need to check 'days <= 6' here.